hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/utils/columnSearch.tsx

```diff
@@ -69,8 +69,16 @@ class ColumnSearch extends React.PureComponent {
     filterIcon: (filtered: boolean) => (
       <Icon type='search' style={{color: filtered ? '#1890ff' : undefined}}/>
     ),
-    onFilter: (value: string, record: any) =>
-      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase()),
+    onFilter: (value: string, record: any) => {
+      if (record[dataIndex] !== undefined || record[dataIndex] !== null) {
+          return typeof (record[dataIndex]) === typeof {}
+            ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
+            : record[dataIndex].toString().toLowerCase().includes(value.toLowerCase())
+        }
+        else {
+          return;
+        }
```

Comment:
could you fix the indentation here? it is not to easy to understand the if/elses here, so it's better to have it more readable. I'm not sure if my suggestion is correct, it's hard to do it in the comment, but as I see in this file it should be 2 spaces. 
```suggestion
        return typeof (record[dataIndex]) === typeof {}
          ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
          : record[dataIndex].toString().toLowerCase().includes(value.toLowerCase())
      }
      else {
        return;
      }
```
```suggestion
          return typeof (record[dataIndex]) === typeof {}
            ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
            : record[dataIndex].toString().toLowerCase().includes(value.toLowerCase())
        }
        else {
          return;
        }
```

Reply:
@dombizita 
Completed Change of space Indentation.